{"stability":{"unstable":"ðŸ”¬ This is a nightly-only experimental API.  (alloc_error_hook #51245)"},"apiName":"std::alloc::take_alloc_error_hook","apiType":"function","apiDocumentation":{"apiDescription":["Unregisters the current allocation error hook, returning it.","See also the function set_alloc_error_hook.","If no custom hook is registered, the default hook will be returned."]}}